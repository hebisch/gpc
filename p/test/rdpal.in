hi there
abcdefgfedcba
able was i ere i saw elba
madam im adam
the quick brown fox jumps over the lazy dog
