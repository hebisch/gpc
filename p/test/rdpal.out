hi there is not a palindrome
abcdefgfedcba is a palindrome
able was i ere i saw elba is a palindrome
madam im adam is not a palindrome
the quick brown fox jumps over the lazy dog is not a palindrome
